tls/s2n_client_hello.c

```diff
@@ -310,9 +279,63 @@ int s2n_client_hello_recv(struct s2n_connection *conn)
         }
     }
 
+    return 0;
+}
+
+int s2n_server_session_lookup(struct s2n_connection *conn)
```

Comment:
The name is a bit confusing: it talks about "session lookup", but the body of the function does ClientHello processing. Should it be `s2n_process_client_hello`?

Reply:
done